cmd/protoc-gen-gorums/dev/zorums_types_gorums.pb.go

```diff
@@ -590,3 +593,215 @@ func (c *CorrectableStreamResponse) set(reply *Response, level int, err error, d
 		}
 	}
 }
+
+// ZorumsService is the server-side API for the ZorumsService Service
+type ZorumsService interface {
+	Multicast(*Request)
+	MulticastPerNodeArg(*Request)
+	Multicast2(*Request)
+	Multicast3(*Request)
+	Multicast4(*empty.Empty)
+	OrderingQC(*Request) *Response
+	OrderingPerNodeArg(*Request) *Response
+	OrderingCustomReturnType(*Request) *Response
+	OrderingCombo(*Request) *Response
+	OrderingUnaryRPC(*Request) *Response
+	OrderingFuture(*Request) *Response
+	OrderingFuturePerNodeArg(*Request) *Response
+	OrderingFutureCustomReturnType(*Request) *Response
+	OrderingFutureCombo(*Request) *Response
+	Unicast(*Request)
+	Unicast2(*Request)
+}
+
+func (s *GorumsServer) RegisterZorumsServiceServer(srv ZorumsService) {
+	s.srv.handlers[multicastMethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		if err != nil {
+			return nil
+		}
+		srv.Multicast(req)
+		return nil
+	}
+	s.srv.handlers[multicastPerNodeArgMethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		if err != nil {
+			return nil
+		}
+		srv.MulticastPerNodeArg(req)
+		return nil
+	}
+	s.srv.handlers[multicast2MethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		if err != nil {
+			return nil
+		}
+		srv.Multicast2(req)
+		return nil
+	}
+	s.srv.handlers[multicast3MethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		if err != nil {
+			return nil
+		}
+		srv.Multicast3(req)
+		return nil
+	}
+	s.srv.handlers[multicast4MethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(empty.Empty)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		if err != nil {
+			return nil
+		}
+		srv.Multicast4(req)
+		return nil
+	}
+	s.srv.handlers[orderingQCMethodID] = func(in *ordering.Message) *ordering.Message {
+		req := new(Request)
+		err := proto.UnmarshalOptions{AllowPartial: true, DiscardUnknown: true}.Unmarshal(in.GetData(), req)
+		// TODO: how to handle marshaling errors here
```

Comment:
Should we add `error` to the interface methods, to allow these errors to be reported?